AHK plugin/Sdl.Community.AhkPlugin/Interface/ISelectFolderDialogService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Sdl.Community.AhkPlugin.Interface
+{
+	public interface ISelectFolderDialogService
```

Comment:
Is this service used on the UI? Does we have on some button the action to select a folder?